internal/gamepad/gamepad_xbox_windows.go

```diff
@@ -225,12 +227,11 @@ func (n *nativeGamepadXbox) isButtonPressed(button int) bool {
 	// Use XInput's trigger dead zone.
 	// See https://source.chromium.org/chromium/chromium/src/+/main:device/gamepad/public/cpp/gamepad.h;l=22-23;drc=6997f8a177359bb99598988ed5e900841984d242
 	// TODO: Integrate this value with the same one in the package gamepaddb.
```

Comment:
Move this comment to an appropriate place.

Reply:
Done, and also did that integrating.